.circleci/config.yml

```diff
@@ -0,0 +1,82 @@
+---
+version: 2.1
+
+orbs:
+    slack: circleci/slack@3.4.2
+
+executors:
+  builder:
+    docker:
+      - image: circleci/php:latest
+  publisher:
+    docker:
+      - image: gcr.io/google.com/cloudsdktool/cloud-sdk:alpine
+        auth:
+          username: _json_key
+          password: $GCLOUD_GCR_ADMIN_KEY
+
+jobs:
+  build:
+    executor: builder
+    working_directory: ~/prestashop
+    steps:
+      - checkout
+      - run:
+          name: Build Package
+          command: sh packlink-build.sh ${CIRCLE_TAG}
```

Comment:
what happens in branches? no tag..

Reply:
must be only build in releases, no branches